lib/p2p/Peer.ts

```diff
@@ -97,6 +107,17 @@ class Peer extends EventEmitter {
     }
   }
 
+  public get info(): PeerInfo {
+    return {
+      socketAddress: this.id,
+      nodePubKey: this.handshakeState ? this.handshakeState.nodePubKey : undefined,
+      direction: this.direction,
+      supportedPairs: this.handshakeState ? this.handshakeState.pairs : undefined,
+      xudVersion: this.handshakeState ? this.handshakeState.version : undefined,
+      timeConnected: (this.connectTime - Date.now()) / 1000,
```

Comment:
Should be something like: `Math.round((Date.now() - this.connectTime) / 1000)` so that the value is positive and an integer.